.claude/skills/implement/SKILL.md

```
@@ -1,81 +1,58 @@
+   - 없으면 현재 브랜치명에서 이슈 번호를 파싱해 사용하라 (`git branch --show-current`).
+2. 계획서를 Read로 읽어라.
+   - 파일이 없으면 "계획서가 없습니다. 먼저 `write-plan` 스킬을 실행하세요"를 알리고 중단하라.
+   - "결정 필요"에 미체크 항목이 남아있으면, 이는 치명적이므로 사용자에게 확정을 요청하고 반영(계획서 Edit) 전까지 구현을 시작하지 마라.
 
-1. `.claude/rules/layer-convention.md`를 Read로 읽어라
-2. `.claude/rules/common-code-convention.md`를 Read로 읽어라
-3. DB 변경이 필요하면 `.claude/rules/database.md`를 Read로 읽어라
-
-> 다음 Phase 조건: 관련 컨벤션 파일을 모두 읽었을 때
+> 다음 Phase 조건: 계획서를 읽었고 미결정 항목이 없을 때
 
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)